common/openapi.yml

```diff
@@ -153,6 +153,11 @@ paths:
         - $ref: '#/components/parameters/Id'
         - $ref: '#/components/parameters/WorkspaceId'
       operationId: exportDataset
+      requestBody:
+        description: A empty json request body to avoid Content-Length errors
+        content:
+          application/json: { }
```

Comment:
If this does fix the problem, maybe we should move `workspaceId` to the body so it will always be present?

Reply:
Thoughts on your earlier comment? https://github.com/DataBiosphere/terra-data-catalog/pull/113#discussion_r930521292